schemas/agent.yml

```diff
@@ -23,6 +22,14 @@
 
       example: 6.0.0-rc2
 
+    - name: build.original
+      level: core
+      type: keyword
+      description: >
+        Extended build information for the agent.
+
+      example: metricbeat version 7.6.0 (amd64), libbeat 7.6.0 [6a23e8f8f30f5001ba344e4e54d8d9cb82cb107c built 2020-02-05 23:10:10 +0000 UTC]
```

Comment:
I think we should be explicit here that no format is mandated. So a few things:

- [ ] copy the current first sentence to populate `short:` as is, I think it's great as a one liner.
- [ ] expand `description:` with a second sentence (or para) to the effect that no format is mandated, it's up to the data source to populate with anything they have / need.